### ISSUE TITLE
pwclient info fails with "the JSON object must be str, bytes or bytearray, not dict"

### ISSUE THREAD

**Issue by jacob-keller** [getpatchwork/pwclient#21]
I tried to use pwclient, but it fails with "the JSON object must be str, bytes or bytearray, not dict"

```
$ pip install pwclient
$ pip show pwclient
Name: pwclient
Version: 2.5.0
Summary: The command-line client for the Patchwork patch tracking tool
Home-page: https://github.com/getpatchwork/pwclient
Author: Nate Case
Author-email: ncase@xes-inc.com
License: GPL v2
Location: /home/jekeller/.local/lib/python3.10/site-packages
Requires:
Required-by:

$ cat ~/.pwclientrc
[options]
default = kernel

[kernel]
url = https://patchwork.kernel.org/api/
backend = rest
username = jekeller
token = <redacted>

$ pwclient info 13185597
the JSON object must be str, bytes or bytearray, not dict
```

**Comment by jacob-keller** [comment]
Turns out this was due to the trailing slash in the URL, which makes the pwclient use URLs such as

https://patchwork.kernel.org/api//patches/13185597

These fail, and return a {} instead of a string, resulting in the weird error. The error handling could be improved to make this more obvious.

**Comment by stephenfin** [comment]
Thanks for the report. Fixed in 2.6.0.
